scania-digital-design-system/tegel
# fix(chip): no double click when switching to active for type radio

**Describe pull-request**  
Setting chip to active when type radio should not require to double click. 

**Solving issue**  
Fixes: [CDEP-2995](https://tegel.atlassian.net/browse/CDEP-2995)

**How to test**  
_Add description how to test if possible_
1. Go to Storybook
2. Check in Chip component, and change to type radio, setting to active should not require double click

[CDEP-2995]: https://tegel.atlassian.net/browse/CDEP-2995?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
